src/components/TranslationLeaderboard.js

```diff
@@ -0,0 +1,245 @@
+// Libraries
+import React, { useState } from "react"
+import styled from "styled-components"
+
+// Components
+import Emoji from "./Emoji"
+import { ButtonSecondary } from "./SharedStyledComponents"
+import Translation from "./Translation"
+
+// Data
+import monthData from "../data/translation_reports/month_data.json"
```

Comment:
Minor nitpick & I don't believe this is actually stated anywhere but we've used dashes vs. underscores for directory & file names.